.github/workflows/insitu.yml

```
@@ -68,3 +68,55 @@ jobs:
           *.png
           conduit_*
         if-no-files-found: error
+
+  catalyst:
+    name: Catalyst
+    runs-on: ubuntu-20.04
+    if: github.event.pull_request.draft == false
+    env:
+      CXX: g++
+      CC: gcc
+      CMAKE_PREFIX_PATH: "/opt/conduit:/opt/catalyst"
+      CATALYST_DEBUG: 1
+      CATALYST_IMPLEMENTATION_PATHS: /opt/paraview/lib/catalyst
```

To avoid oversubscription of the 2-4 CI cores we have on GH runners:
```suggestion
      CATALYST_IMPLEMENTATION_PATHS: /opt/paraview/lib/catalyst
      OMP_NUM_THREADS: 1
```
(oops, maybe the other two in situ tests need this, too)